pilot/pkg/features/pilot.go

```diff
@@ -406,4 +406,10 @@ var (
 	PilotEnableLoopBlockers = env.RegisterBoolVar("PILOT_ENABLE_LOOP_BLOCKER", true,
 		"If enabled, Envoy will be configured to prevent traffic directly the the inbound/outbound "+
 			"ports (15001/15006). This prevents traffic loops. This option will be removed, and considered always enabled, in 1.9.").Get()
+
+	PilotJwtPubKeyRefreshInterval = env.RegisterDurationVar(
+		"PILOT_JWTPUBKEY_REFRESH_INTERVAL0",
+		20*time.Minute,
```

Comment:
Seems far too low as a default. 1 week maybe - do we have data on how often typical OIDC are refreshing their root ?

Reply:
That's not changed in the PR, there is no common standard around the default, some provider (e.g. Firebase) has a expiration interval of several hours.